src/app/styles/constants/stylesCore.js

```diff
@@ -0,0 +1,6 @@
+export const stylesCore = {
```

Comment:
🟢 can't these few styles be placed in button styles? as plain consts

Reply:
Thanks for comment! Done.